Dockerfile

```
@@ -12,34 +12,41 @@ WORKDIR /nativefier
 # Add sources
 COPY . .
 
+# Give everything to node
+RUN chown -R node:node /nativefier \
+    && chown -R node:node /tmp
+
+# Use node (1000) as default user not root
+USER node
+
 # Fix line endings that may have gotten mangled in Windows
 RUN find ./icon-scripts ./src ./app -type f -print0 | xargs -0 dos2unix
 
-# Build nativefier and link globally
-WORKDIR /nativefier/app
-RUN npm install
-WORKDIR /nativefier
-
-# Install (note that we had to manually install in `app` before, as `prepare` won't run as root)
-# Also, running tests, to ensure we don't Docker build & publish broken stuff
-RUN npm install && npm run build && npm test && npm link
+# Setup a global packages location for "node" user so we can npm link
+RUN mkdir ~/.npm-packages \
```

In a normal user machine, a hidden dir makes total sense (that's what I do on my machine too), but in a Docker, there's no point to it.

```suggestion
RUN mkdir ~/npm-packages \
```

(... plus needs edit in the following lines, but GitHub won't let me)